ros/lib/models.py

```diff
@@ -34,6 +34,11 @@ class System(db.Model):
     id = db.Column(db.Integer, primary_key=True)
     account_id = db.Column(db.Integer)
     inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
+    display_name = db.Column(db.String(100))
```

Comment:
do we want the display name to be unique?

Reply:
No, it may happen that two user account has the system with the same display name. So it cannot be unique.